CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+## 0.19.0 - TBD
+
+### Features
+
+Redis Sensu transport, providing a built-in alternative to the default
```

Comment:
I'll drop the unnecessary comma, but I think providing works/sounds better in this case, no?